Library/Homebrew/cmd/certs.rb

```diff
@@ -0,0 +1,50 @@
+#:  * `certs` [`--force-curl`]:
+#:    You should really write more documentation up here Dom.
+
+module Homebrew
+  BREWED_CURL = Pathname.new(HOMEBREW_PREFIX/"opt/curl")
+  BREWED_OPENSSL = Pathname.new(HOMEBREW_PREFIX/"opt/openssl")
+  BREWED_LIBRESSL = Pathname.new(HOMEBREW_PREFIX/"opt/libressl")
+  BREWED_GNUTLS = Pathname.new(HOMEBREW_PREFIX/"opt/gnutls")
```

Comment:
I would avoid all these paths and `#exist?` method in the blow. Instead, I think using formula is better. So it would be like `safe_system "brew", "postinstall", "openssl" if Formula["openssl"].installed?`